test/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring.Tests/Linux/LinuxUtilizationParserCgroupV1Tests.cs

```diff
@@ -8,16 +8,14 @@
 using System.IO;
 using System.Threading.Tasks;
 using Microsoft.Shared.Pools;
-using Microsoft.TestUtilities;
 using Moq;
 using Xunit;
 
 namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring.Linux.Test;
 
-[OSSkipCondition(OperatingSystems.Windows | OperatingSystems.MacOSX, SkipReason = "Linux specific tests")]
```

Comment:
Same: it's Linux only: so use `[PlatformSpecific(TestPlatforms.Linux)]` for the entire class and don't use `[LinuxOnlyTheory]`